laravel/app/Http/Controllers/PostController.php

```diff
@@ -51,20 +52,30 @@ public function store(Request $request)
          ]);
 
          $post = new Post();
-
          //画像はbase64で受け取っている。
          if(strpos($request->image,'data:image/png;base64') !== false){
              $image = base64_decode(str_replace(' ', '+',str_replace('data:image/png;base64,', '', $request->image)));
              $post->image = hash('sha256',Str::random(20).time()).'.'.'png';
              File::put(storage_path('app/public/image/PostImage'). '/' . $post->image, $image);
          }else{
-             return back()->with('error', '選択できるのは画像のみです。');
+             return back()->with('error', '画像が選択されていないか、画像以外が選択されています。');
          }
          $post->title = $validatedData['title'];
          $post->comment = $validatedData['comment'];
          $post->user_id = Auth::id();
          $post->save();
 
+         //文字列をタグの形式に置換して、dbに保存する
+         $substitute_tag = explode(',',str_replace('，',',',$request->tags));
+         foreach ($substitute_tag as $comment) {
```

Comment:
tagなのにcommentという変数名は混乱を招くので修正しましょう🐱
```suggestion
         foreach ($substitute_tag as $tag) {
```